library/src/main/java/com/bumptech/glide/module/ChildGlideModule.java

```diff
@@ -0,0 +1,21 @@
+package com.bumptech.glide.module;
+
+import android.content.Context;
+import com.bumptech.glide.Registry;
+
+/**
+ * Registers a set of components to use when initializing Glide within an app when
+ * Glide's annotation processor is used.
+ *
+ * <p>Any number of ChildGlideModules can be contained within any library or application.
+ *
+ * <p>ChildGlideModules are called in no defined order. If ChildGlideModules within an application
+ * conflict, {@link RootGlideModule}s can use the {@link com.bumptech.glide.annotation.Excludes}
+ * annotation to selectively remove one or more of the conflicting modules.
+ */
+public abstract class ChildGlideModule implements RegistersComponents {
```

Comment:
Class so that they're more extensible. Same logic that the framework is using. If you ever want to change or add a method, it's possible without breaking everyone.